lib/Service/Social/MastodonProvider.php

```diff
@@ -104,14 +104,18 @@ protected function getProfileIds($contact):array {
 					if (isset($masto_user_server)) {
 						try {
 							[$masto_user, $masto_server] = $masto_user_server;
-							# search for user on Mastodon
-							$search = $masto_server . '/api/v2/search?q=' . $masto_user . '@' . parse_url($masto_server)["host"];
-							$result = $this->httpClient->get($search);
+							# search for user webfinger
```

Comment:
@tcitworld I am astonished this is part of my pull request. thought we had done this modification a long time ago (see https://github.com/nextcloud/contacts/issues/3210#issuecomment-1413360452). Or did I only try it out locally and never submitted? Can you confirm it is better this way?

Reply:
can you help me adapt the tests? I think that might be the reason I did not send out a PR.

Reply:
Tests are up to date now again.